ckanext/restricted_access/middleware.py

```diff
@@ -14,6 +14,8 @@
 
 
 log = logging.getLogger(__name__)
+ERROR_CODE = conf.get_restricted_paths_error_code()
+ERROR_MESSAGE = conf.get_restricted_paths_error_message()
```

Comment:
You don’t need a runtime constant in middleware.py:
1. In future we want to be able to change CKAN config options on the fly, without restarting the server. In this case, having a runtime constant would be semantically wrong.
2. You are using it only once